### DISCUSSION TITLE
Add button tootips in viewer menu

### DISCUSSION THREAD

**Discussion by m1ga** [rustdesk/rustdesk#5691] [Feature Request] [answered]
Would be really nice to have tooltips for those buttons:
![Screenshot_20230914_121630](https://github.com/rustdesk/rustdesk/assets/4334997/cf5679be-59d9-4b35-a6f7-4fdc3c86ec04)

On my Fedora app I don't see any if I hover them with the mouse. Icons in the main view like "last sessions", "favorites" have tooltips.

**Comment by rustdesk** [comment, marked as answer]
https://github.com/rustdesk/rustdesk/pull/5334

**Reply by m1ga** [reply]
oh sorry about that. My Linux machine was a version behind (1.2.1) with 1.2.2 I can see them but it takes almost 2 seconds to show them. Even the icons in the main window now take longer to show the tooltip. Is saw that you've raised the timeout in another PR. Maybe it is a tiny bit too slow now :)

https://github.com/rustdesk/rustdesk/assets/4334997/1db53e07-96a7-4a34-adb6-269986574f4a

normal tooltips show up quicker:


https://github.com/rustdesk/rustdesk/assets/4334997/0e847d29-a267-4782-b10c-6f5130d64e57
